react-carousel-component/nimmys-app/src/Container.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from 'react';
+import { useState } from 'react';
+
+export default function Container({ imgs }) {
+  const [player, setPlayer] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(
+      () => setPlayer((player + 1) % imgs.length),
```

Comment:
Avoiding this code duplication can be a challenge! I'll show you how when I walk through my code solution.